repos/system_upgrade/el7toel8/actors/mytestdialog/actor.py

```diff
@@ -0,0 +1,44 @@
+import time
+from leapp.actors import Actor
+from leapp.dialogs import Dialog
+from leapp.dialogs.components import BooleanComponent
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.stdlib import api
+
+
+class MyCustomQuestion(Actor):
+    """
+    Confirm suggested authselect call from AuthselectScanner.
+
+    AuthselectScanner produces an Authselect model that contains changes
+    that are suggested based on current configuration. This actor will
+    ask administrator for confirmation and will report the result.
+    """
+
+    name = 'time_check'
+    consumes = ()
+    produces = ()
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+    dialogs = (Dialog(scope='time_check',
+                      reason='Confirmation',
+                      components=(BooleanComponent(key='confirm',
+                                                   label='Is it lunchtime yet?',
+                                                   default=True,
+                                                   description='',
+                                                   reason='I am hungry.'),)),)
+
+    def process(self):
+        dialog = self.dialogs[0]
+
+        dialog.components[0].label += " {}".format(time.ctime())
+
+        result = self.get_answers(dialog).get('confirm')
+
+        api.current_logger().debug('Answer to whether there is lunch: {}'.format(result))
+
+        if result:
+            api.current_logger().warning('LUNCHTIME!')
+        elif result is False:
+            api.current_logger().warning('No lunch yet.')
+        else:
+            api.current_logger().warning('WHAT EVEN IS LUNCH? {}'.format(result))
```

Comment:
@dkubek  :-D

Reply:
Whoops! Let's save this functionality maybe for a later release :joy: